README.md

```diff
@@ -45,6 +45,7 @@ Here are some commands you can run:
 | yarn build-ci               | Run the `build-all` and `bundlesize` npm scripts.                                                                                                                                               |
 | yarn bundlesize             | Run [bundlesize][] to check the generated AMO bundle sizes. [Building AMO is required first](#building-and-running-services).                                                                   |
 | yarn disco                  | Start the dev server (for Discovery Pane) using data from the dev server (https://addons-dev.allizom.org/)                                                                                      |
+| yarn disco:https            | Same as `disco` but with HTTPS, available at: https://example.com:3000/. [Read about setting up this environment](docs/moz-addon-manager.md#developing-with-a-local-https-server-recommended)   |
```

Comment:
We seem to be pretty inconsistent in this file about including a period (.) at the end of each sentence. I think we should choose one and be consistent. As some of these descriptions contain multiple sentences, I think it probably makes sense to just put a period at the end of all of them.

Reply:
adding this to my other patch related to documentation, good idea 👍